core/src/main/scala/astraea/spark/rasterframes/expressions/package.scala

```diff
@@ -93,6 +93,10 @@ package object expressions {
     registry.registerExpression[DataCells]("rf_data_cells")
     registry.registerExpression[NoDataCells]("rf_no_data_cells")
     registry.registerExpression[IsNoDataTile]("rf_is_no_data_tile")
+    registry.registerExpression[Exists]("rf_exists")
+    registry.registerExpression[Exists]("rf_any")
+    registry.registerExpression[ForAll]("rf_for_all")
+    registry.registerExpression[ForAll]("rf_all")
```

Comment:
Ditto. Also, we're introducing a name (`all`) that doesn't exist in the JVM library.